x-pack/legacy/plugins/infra/server/routes/index_pattern.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { InfraBackendLibs } from '../lib/infra_types';
+import { InfraWrappableRequest } from '../lib/adapters/framework';
+import { internalInfraFrameworkRequest } from '../lib/adapters/framework';
+
+const INDEX_PATTERN_ID = 'metricbeat-*';
```

Comment:
For now let's change this to `_internal_metrics_explorer_tsvb` to signal our intentions better.

Reply:
Let's also leave some comments near the top of this file explaining why we need to create an index pattern.